src/material-examples/tree-flat/tree-flat-example.html

```diff
@@ -0,0 +1,9 @@
+<cdk-tree [dataSource]="dataSource" [treeControl]="treeControl">
+  <cdk-tree-node *cdkTreeNodeDef="let node" cdkTreeNodePadding class="demo-tree-node">
+    <a [attr.aria-label]="'toggle ' + node.filename" cdkTreeNodeToggle class="demo-tree-toggle">
```

Comment:
Anchor elements should only be used for navigation (should always have an `href`). For actions, you should use a styled `<button>`.